modules/ROOT/pages/autocorrect.adoc

```diff
@@ -0,0 +1,93 @@
+= Spelling Autocorrect plugin
+:navtitle: Spelling Autocorrect
+:description: Autocorrect common spelling errors.
+:keywords: autocorrect, capitalization, misprint, spell, spelling, spellcheck, typo, typographical error
+:pluginname: Spelling Autocorrect
+:plugincode: autocorrect
+:pluginminimumplan: tierthree
+
+
+include::partial$misc/admon-premium-plugin.adoc[]
+
+include::partial$misc/admon-requires-6.2v.adoc[]
+
+The {pluginname} plugin allows {productname} to autocorrect a specified set of spelling errors, common typos, and capitalization errors, as they are being typed.
+
+== Interactive Example
+
+liveDemo::autocorrect[]
+
+include::partial$misc/purchase-premium-plugins.adoc[]
+
+== Basic setup
+
+[source,js]
+----
+tinymce.init({
+  selector: 'textarea',  // change this value according to your HTML
+  plugins: 'autocorrect'
+});
+----
+
+[[usage-notes]]
+== Usage notes
+
+The {pluginname} plugin currently has the following limitations:
+
+. English is the only supported language.
+
+. Capitalization and autocorrect can both be turned off, but specific autocorrections cannot.
++
+{pluginname} can be entirely turned off, but there is, currently, no way of instructing the plugin to ignore a particular string that it considers a mis-spelling.
++
+For example, if *cna* (which the {pluginname} plugin treats as a mis-typing of *can*) is a desired string in a particular use-case, turning {pluginname} off is the currently recommended setting.
```

Comment:
(discussion): use backtick or something else to make it clear that this is intentional
```suggestion
For example, if `+cna+` (which the {pluginname} plugin treats as a mis-typing of *can*) is a desired string in a particular use-case, turning {pluginname} off is the currently recommended setting.
```